provider_shopper/test/cart_widget_test.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:provider_shopper/models/cart.dart';
+import 'package:provider_shopper/models/catalog.dart';
+import 'package:provider_shopper/screens/cart.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
```

Comment:
This call shouldn't be necessary. If you're seeing a failure without it, that's something we'd want to know about.